charts/bitwarden-crd-operator/crds/bitwarden-secrets.yaml

```diff
@@ -41,11 +41,20 @@ spec:
                           - secretName
                 id:
                   type: string
+                item_name:
+                  type: string
+                collection_path:
+                  type: string
                 namespace:
                   type: string
                 name:
                   type: string
               required:
-                - id
                 - namespace
                 - name
+              oneOf:
+              - required:
+                  - id
+              - required:
+                  - item_name
```

Comment:
Should be camelCase as you wrote in the example

Reply:
> I guess this needs more testing (...)

I will recheck what cab be possible wrong

> as a last point i would put the periodic sync in a dedicated function  (...)

Yes please, do it. I will remove it from my MR.